vNext/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IChannelControls.ts

```diff
@@ -27,9 +27,10 @@ export interface IChannelControls extends ITelemetryPlugin {
     /**
      * Flush to send data immediately; channel should default to sending data asynchronously
      * @param async: send data asynchronously when true
+     * @param isBeaconSender: send data through beacon when true
      * @param callBack: if specified, notify caller when send is complete 
      */
-    flush(async: boolean, callBack?: () => void): void;
+    flush(async: boolean, isBeaconSender?: boolean, callBack?: () => void): void;
```

Comment:
Verify that this interface isn't used anywhere else here or in other SDKs. 
@hectorhdzg Is this used anywhere that you know of?